python/modules/IcePy/Util.cpp

```diff
@@ -885,10 +881,9 @@ IcePy::callMethod(PyObject* method, PyObject* arg1, PyObject* arg2)
         {
             return nullptr;
         }
-        Py_XINCREF(arg1);
-        PyTuple_SET_ITEM(args.get(), 0, arg1);
-        Py_XINCREF(arg2);
```

Comment:
Here Py_XINCREF does't make sense because you cannot pass nullptr to PyTuple_SET_ITEM.